src/components/WalletUnlock/DeterministicAccountTable.tsx

```diff
@@ -321,100 +305,108 @@ const DeterministicTable = ({
                               fromTokenBase(
                                 new BN(account.balance.toString()),
                                 asset.decimal
                               ).toString()
                             ).toFixed(4)
-                            : '0.0000'
-                        }
-                      />
-                    </Typography>
-                    <Typography>{asset.ticker}</Typography>
-                  </ValueContainer>
-                </MobileColumn>
-                <LinkContainer>
-                  <Tooltip tooltip={'View on Etherscan'}>
-                    <LinkOut
-                      link={
-                        network.blockExplorer
-                          ? network.blockExplorer.addressUrl(account.address)
-                          : `https://ethplorer.io/address/${account.address}`
+                          : '0.0000'
                       }
                     />
-                  </Tooltip>
-                </LinkContainer>
-              </Row>
-            ))}
-            <GenerateAddressButton
-              onClick={generateFreshAddress}
-              disabled={!isComplete || freshAddressIndex >= DEFAULT_GAP_TO_SCAN_FOR}
-            >
-              <Icon type="add" />
-
-              {!isComplete ? (
-                <Tooltip tooltip={<Trans id="DETERMINISTIC_WAIT_FOR_SCAN" />}>
-                  <STypography>
-                    <Trans id="DETERMINISTIC_GENERATE_FRESH_ADDRESS" />
-                  </STypography>
-                </Tooltip>
-              ) : freshAddressIndex >= DEFAULT_GAP_TO_SCAN_FOR ? (
-                <Tooltip
-                  tooltip={
-                    <Trans
-                      id="DETERMINISTIC_CANT_GENERATE_MORE"
-                      variables={{ $number: () => DEFAULT_GAP_TO_SCAN_FOR }}
-                    />
-                  }
-                >
-                  <STypography>
-                    <Trans id="DETERMINISTIC_GENERATE_FRESH_ADDRESS" />
-                  </STypography>
+                  </Typography>
+                  <Typography>{asset.ticker}</Typography>
+                </ValueContainer>
+              </MobileColumn>
+              <LinkContainer>
+                <Tooltip tooltip={'View on Etherscan'}>
+                  <LinkOut
+                    link={
+                      network.blockExplorer
+                        ? network.blockExplorer.addressUrl(account.address)
+                        : `https://ethplorer.io/address/${account.address}`
+                    }
+                  />
                 </Tooltip>
-              ) : (
-                    <STypography>
-                      <Trans id="DETERMINISTIC_GENERATE_FRESH_ADDRESS" />
-                    </STypography>
-                  )}
-            </GenerateAddressButton>
-          </Body>
-        )}
+              </LinkContainer>
+            </Row>
+          ))}
+          <GenerateAddressButton
```

Comment:
What has changed here at the bottom? Just whitespace? The diff is hard to read 😆

Reply:
Prettified :)